tests/test_company_api.py

```
@@ -43,3 +43,65 @@ def test_public_doctor_endpoint(self, client):
+    "request: Field required" (surfaced in the UI as
+    "Could not create company: request: Field required").
+    """
+
+    def test_thunk_request_param_is_annotated_as_Request(self):
+        """The auth thunks must annotate `request` as `Request`, otherwise
+        FastAPI demands a body field named "request"."""
+        import inspect
+        try:
+            from fastapi import Request
+            from backend import company_api
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)